### ISSUE TITLE
Wondering if this extension is being maintained/developed at all?

### ISSUE THREAD

**Issue by Upperholme** [compucorp/uk.co.compucorp.civicrm.contactcalendar#4]
And if there are plans to ensure it is compatible with current versions of CiviCRM (4.6, 4.7)?
Thanks for any response.

**Comment by mehalforc** [comment]
If you want to wait until I update mine to version 3, you can try it. It should work with 4.7... Not sure about 4.6. My version 2 beta has an issue. I will update repository by end of the week.

**Comment by Upperholme** [comment]
That's great, thanks. I'll check it out next week.

**Comment by mehalforc** [comment]
Yeah, what is cool with mine is... You can color it and hide/show specific ones. Just modify the *.tpl code. And it is click-able to view from the calendar. :)

**Comment by jamienovick** [comment]
Hi Mehal, if you can send PR we can merge it in, otherwise happy for you to take this forward. Best Jamie

**Comment by mehalforc** [comment]
I will update the repository and resend the PR sometime today. Don't use the one I sent awhile back. It has an issue.

**Comment by mehalforc** [comment]
PR sent, but test it first as I haven't tested it on the latest version of CiviCRM. :)

**Comment by colemanw** [comment]
FYI there is a new [Calendar Display for SearchKit](https://lab.civicrm.org/ayduns/calendarskdisplay) under development, which would be an alternative to this extension.
